demonstrations/tutorial_barren_gadgets.py

```diff
@@ -0,0 +1,332 @@
+r"""
+Perturbative Gadgets for Variational Quantum Algorithms  
+==========================================
+
+*Author: Simon Cichy*
+
+In this tutorial, we will explore the application of perturbative gadgets in 
+variational quantum algorithms to outgo the issue of cost function dependent
+barren plateaus, as proposed in Ref.[#cichy2022]
+
+Some context
+------------
+
+If you are not familiar yet with the concept of barren plateaus, I recomend you
+first check out these other tutorials: 
+- https://pennylane.ai/qml/demos/tutorial_barren_plateaus.html
+- https://pennylane.ai/qml/demos/tutorial_local_cost_functions.html  

```

Comment:
```suggestion
If you are not familiar yet with the concept of barren plateaus, I recommend you
first check out the demonstrations on :doc:`barren plateaus </demos/tutorial_barren_plateaus>`
and :doc:`avoiding barren plateaus with local cost functions </demos/tutorial_local_cost_functions>`.
```

Using this style of local link ensure that the tutorial is not affected if we ever change the URL of the website :)